breakout_3_Observability-with-AI/breakout_3_answers/1.2-grafana-assistant.md

```diff
@@ -0,0 +1,14 @@
+## Grafana AI
+Q: What's the LCP over the past few hours? Any bad pages?
+
+A: 
+- LCP Events: Consistently recording ~60 LCP events per minute, indicating stable page activity
+- Performance Rating: Mostly "good" ratings across different global probes
+- Response Times:
+    - Best: Calgary probe (~936ms average) ✅
+    - Moderate: Paris probe (~1,280ms average) 🟡
+    - Concerning: CapeTown probe (~2,598ms average, "needs-improvement" rating) ⚠️
```

Comment:
This is based on synthetic probes - the question is designed for frontend o11y data not for synthetic data. We might need to give this context or ask it as a follow up question

Reply:
Good to know, will tweak the prompt a little bit.